### PR TITLE
net-wireless/*: EAPI5--, EAPI7++, drop old

### PR DESCRIPTION
**Pull request by a17r**
Assorted fixes.

### CODE REVIEW

**Comment by ZeroChaos-** [comment]
this cripples my workflow entirely for purposes which I don't see.  I don't mind the eapi bumps but removing my if/else blocks to allow live ebuilds to be renamed to stable ebuilds is not welcome.  especially with the radio stuff this is absolutely required for me to have any sanity as most of these projects don't make releases and I have to track git and switch between often.

NACK

**Comment by ZeroChaos-** [comment]
keeping this open for back and forth.  per discussion in IRC I will allow dropping the KEYWORDS="" lines

**Comment by a17r** [comment]
Rebased, made ebuilds look more unified and re-added conditional single `KEYWORDS` line anywhere you had it already before.

**Comment by ZeroChaos-** [comment]
please don't remove bladerf-2017.12_rc1.ebuild or anything that depends on it.  this is the last version that supports libbladerf v1 api and the v2 api has only seen one release.

File: net-wireless/gr-foo/gr-foo-9999.ebuild [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
 if [[ ${PV} == 9999* ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/bastibl/gr-foo.git"
+else
 	KEYWORDS=""
-#else
-#	SRC_URI=""
-#	KEYWORDS=""
 fi
+inherit cmake-utils python-single-r1
```
Comment (ZeroChaos-) [inline net-wireless/gr-foo/gr-foo-9999.ebuild]: it is my understanding that inherit is supposed to be as close to the top as possible

Comment (a17r) [reply net-wireless/gr-foo/gr-foo-9999.ebuild]: I think it's fine, with `git-r3` there's an inherit line very close to the top anyway and the whole block belongs together.

**Comment by ZeroChaos-** [comment]
Thanks, please merge this after the CI passes it.  :+1: 
ACK

**Pull request closed without merging**